wasp/src/main/java/com/orhanobut/wasp/WaspImage.java

```diff
@@ -64,6 +70,50 @@ void load() {
         imageHandler.load();
     }
 
+    public void logWaspImageRequest() {
+        switch (logLevel) {
+            case ALL:
+                Logger.d("---> IMAGE REQUEST " + url);
+                Logger.d("Crop - " + cropCenter);
+                Logger.d("Fit - " + fit);
+                if (size != null) {
+                    Logger.d("Size - Width: " + size.getWidth() + " | Height: " + size.getHeight());
+                }
+                Logger.d("---> END");
+                break;
+        }
```

Comment:
should we add default case here? which we notify that the method called but nothing happened